src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -690,9 +647,9 @@ mod tests {
 
     #[test]
     fn anchor_generation() {
-        assert_eq!(id_from_content("## `--passes`: add more rustdoc passes"),
+        assert_eq!(utils::id_from_content("## `--passes`: add more rustdoc passes"),
```

Comment:
Now that the `id_from_content()` function has been moved to the `utils` module it may make sense to relocate these tests as well. I'm also curious to see how the ID generation will go when your header includes emojis, bold, or bits of code.